uber/templates/registration/menu.html

```diff
@@ -1,5 +1,5 @@
 {% if c.HAS_WATCHLIST_ACCESS %}
-{% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "staffers" "Return to Staffer List" attendee.staffing "watchlist?attendee_id={id}" "Watch List Entry" attendee.banned "index" "Return to Attendee List" True %}
+    {% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "watchlist?attendee_id={id}" "Watch List Entry" attendee.banned "index" "Return to Attendee List" c.PRE_CON "index?search_text={id}" "Return to Attendee List" c.AT_THE_CON "index" "Return to Attendee List" c.POST_CON %}
 {% else %}
-{% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "staffers" "Return to Staffer List" attendee.staffing "index" "Return to Attendee List" True %}
-{% endif %}
\ No newline at end of file
+    {% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "index" "Return to Attendee List" c.PRE_CON "index?search_text={id}" "Return to Attendee List" c.AT_THE_CON "index" "Return to Attendee List" c.POST_CON %}
+{% endif %}
```

Comment:
Every time I do anything with the ``nav_menu`` tag, I remember that we need to replace it with something better.  Ah well, low priority.